torchrl/data/datasets/minari_data.py

```diff
@@ -539,3 +565,51 @@ def _patch_info(info_td):
     if not source.is_empty():
         val_td_sel.update(source, update_batch_size=True)
     return val_td_sel
+
+
+def patch_nontensor_data_to_stack(tensordict: TensorDictBase):
```

Comment:
Make private? I don't think we want to commit that we'll keep these for eternity